### PR TITLE
Migrate Installations to GHA

### PR DESCRIPTION
**Pull request by paulb777**
Also delete some additional unneeded travis-specific code.

### CODE REVIEW

File: scripts/setup_quickstart.sh [hunk 1]
```diff
@@ -46,23 +49,5 @@ if check_secrets; then
 
   # Add GoogleService-Info.plist to Xcode project
   ruby ../scripts/info_script.rb "${SAMPLE}"
-
```
Comment (paulb777) [inline scripts/setup_quickstart.sh:49]: Quickstarts are now all run on GHA

File: .travis.yml [hunk 2]
```diff
@@ -114,21 +114,6 @@ jobs:
       script:
         - travis_retry ./scripts/if_changed.sh ./scripts/pod_lib_lint.rb FirebaseMessaging.podspec --platforms=ios
 
-    - stage: test
```
Comment (paulb777) [inline .travis.yml:117]: Already migrated to gha

File: .github/workflows/installations.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      id: secrets
+      run: echo "::set-output name=val::$([[ -z $plist_secret ]] && echo "0" || echo "1")"
+    - name: Build and test
+      run: |
+       export FIS_INTEGRATION_TESTS_REQUIRED=${{ steps.secrets.outputs.val }}
+       export FIS_ALLOWS_INCOMPATIBLE_IID_VERSION="1"
+       scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseInstallations.podspec \
+         --platforms=${{ matrix.target }} --ignore-local-podspecs=FirebaseInstanceID.podspec
+
+  # TODO - Catalyst is disabled because `pod gen` does not have a way to get some dependent pods
```
Comment (maksymmalyhin) [inline .github/workflows/installations.yml:43]: I think we can introduce an additional flag to disable specific tests requiring a specific IID version. We can address this later.

Comment (paulb777) [reply .github/workflows/installations.yml:43]: SG - or do we need to keep testing the old version of IID?

**Pull request merged**